### PR TITLE
[RFC] new relic installation

### PR DESCRIPTION
**Pull request by tkjwa**
Edit: Everything is configured and launched from the infra, only need to install the newrelic package

~~Here is a demo of the installation of NewRelic agent in a NodeJS application.~~

~~Issues:~~
* ~~configuration: the agent requires config from newrelic.js file at the root and/or env vars. The app name must be correctly set so that info from prod don't mix with staging etc ... (airlock-prod, airlock-staging, etc ...)~~
* ~~As the agent is started from the index.ts should we have something like~~
```
{ NEW_RELIC_APP_NAME } = process.env;
if (NEW_RELIC_APP_NAME) { require('newrelic') }
```

### CODE REVIEW

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     * production applications.
+     */
+    level: 'info'
+  },
+  /**
+   * When true, all request headers except for those listed in attributes.exclude
+   * will be captured for all traces, unless otherwise specified in a destination's
+   * attributes include/exclude lists.
+   */
+  allow_all_headers: true,
```
Comment (podefr) [inline newrelic.js:38]: wonder what that means for GDPR and security? should we instead allow no headers and explicitly list the ones we want for observability purposes?

File: src/index.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require('newrelic');
```
Comment (podefr) [inline src/index.ts:1]: import "newrelic"?

File: newrelic.js (hunk 1 above)
Comment (tkjwa) [reply newrelic.js:38]: Let's check the collected data and adjust later depending on what we see in it

File: src/index.ts (hunk 2 above)
Comment (tkjwa) [reply src/index.ts:1]: another possiblity which could lower the chances to send dev data to prod or staging would be to use `node -r newrelic ./dist/index.js` to launch the service in prod/staging
So there is less chance that someone enables it in dev (eg by having a NEW_RELIC env var set to `airlock-prod` ^^) 

But this needs to be defined in the Dockerfile, and the container could also be used for dev ... I don't think it's a good idea to build different versions of the containers, 1 published container should be usable as is in dev, in the community platform or in the prod k8s cluster

**Pull request merged**